themes/hugo-hero-theme/layouts/pages/aboutlayout.html

```diff
@@ -4,9 +4,77 @@
 
 {{ define "main" }}
 {{ partial "hero-image-fullscreen.html" (dict "background" .Params.heroBackground "heading" .Params.heroHeading "subheading" .Params.heroSubHeading "content" .)}}
+<head>
+  <style>
+    .text{
+      padding-left:280px;
+      padding-right:280px;
+      text-align: justify;
+    }
+    .title{
+      text-align:left;
+      padding-left:650px;
+      padding-bottom:30px;
+    }
+    
+    .img{
+      float:right;
+      clear: right;
+    }
 
-<div id="home" class="container pt-8 pb-8 pt-md-10 pb-md-10">
-  <div class="row justify-content-center">
+    .feature{
+      background-blend-mode:lighten;
+      
+    }
+
+    .d-title{
+      margin-top:20px;
+      padding-left:650px;
+      
+    }
+   
+  </style>
+</head>
+
+<body>
+  <br>
+  <br>
+
+  <div class="welcome">
+  <h1 class="title">Our Mission </h1>
+  
+  <!--
+  <img src= "../../images/about-icon.png" alt="computers" class="img" >
+  -->
+  <p class="text"> 
+                  
+    Since being founded in 2010, in association with McGill’s Electrical, 
```

Comment:
This section is pretty long with the sentences being pretty verbose.

I'm wondering if we can either:
1. Shorten this section by employing shorter sentences
2. Cut it down to 1 paragraph and add 3 more `features` to the about section to incorporate the parts we remove

For example, we could add these points:
- `teamwork`
- `learning`
- `hiring` (this is pretty explicit, but that's _okay_, we want sponsors to hire students)